dashboard/lib/services/lesson_import_helper.rb

```diff
@@ -6,6 +6,25 @@ def self.update_lockable_lesson(script_levels, lesson_id)
     [create_activity_with_levels(script_levels, lesson_id, 1)]
   end
 
+  def self.create_lesson_resources(cb_resources, course_version_id)
+    cb_resources.map do |cb_resource|
+      raise unless cb_resource['slug']
+      resource = Resource.find_or_initialize_by(
+        course_version_id: course_version_id,
+        key: cb_resource['slug']
+      )
+      resource.name = cb_resource['name']
+      resource.url = cb_resource['url']
+      resource.type = cb_resource['type']
+      resource.audience = cb_resource['student'] ? 'Student' : 'Teacher'
+      resource.assessment = false
+      resource.download_url = cb_resource['dl_url']
```

Comment:
I noticed that the download_url does not appear to be editable in the resource editor. should I open a Jira item for this field to be editable there?

Reply:
😬 yes, that sounds like a bug

Reply:
https://codedotorg.atlassian.net/browse/PLAT-522

Reply:
Thanks!